sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiCreateOrUpdateContent.Serialization.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ApiManagement;
+
+[CodeGenSerialization(nameof(TermsOfServiceUri), DeserializationValueHook = nameof(DeserializetermsOfServiceUrlValue))]
+public partial class ApiData : IUtf8JsonSerializable, IJsonModel<ApiData>
```

Comment:
actually since we are not implementing these interfaces here, we could omit them